app/scenes/Document/components/Header.tsx

```diff
@@ -264,14 +264,17 @@ function DocumentHeader({
             {!isDeleted && !isRevision && can.listViews && (
               <Collaborators document={document} />
             )}
-            {(isEditing || !user?.separateEditMode) && !isTemplate && isNew && (
-              <Action>
-                <TemplatesMenu
-                  document={document}
-                  onSelectTemplate={onSelectTemplate}
-                />
-              </Action>
-            )}
+            {(isEditing || !user?.separateEditMode) &&
+              !isTemplate &&
+              isNew &&
+              !document.publishedAt && (
```

Comment:
This hides the templates selection menu after a new document is published in the same screen.

This should've been part of the initial PR; I honestly don't know how I missed it 😞

Reply:
Reverting this change actually, as documents are now auto-published in collections and this prevented applying a template.